workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -291,6 +291,7 @@ type ObjectField {
 
 type FieldCommands {
   remove: [JSON!]!
+  edit(requestedTypes: [JsonType]!): [JSON!]!
```

Comment:
Probably want to make `JsonType!`

Reply:
Hmm - still probably want this to be `[JsonType!]!` - this will allow an array of `Enum` or `null` instead of just `Enum`

Reply:
Huh? Must have gotten lost in a squash or revert somewhere 🤷‍♂️.